src/test/java/spoon/test/ctClass/CtClassTest.java

```diff
@@ -95,18 +95,10 @@ public void getConstructor() throws Exception {
 		cons.addParameter(cons.getFactory().createParameter().setType(cons.getFactory().Type().OBJECT));
 		// now that we have changed the signature we can call getConstructors safely
 		assertEquals(4, foo.getConstructors().size());
-		// we cloned the first constructor, so it has the same position, and comes before the 2nd and 3rd constructor
-		assertSame(cons, foo.getTypeMembers().get(1));
```

Comment:
it's better to change the expected value than to remove assertions.

Reply:
now the test should still showcase adding a parameter does not change the constructor order and is still green.